doc/mysql-handler.md

```diff
@@ -41,15 +41,15 @@ CREATE TABLE IF NOT EXISTS `users` (
 
 ```
 
--- Create administration user
+-- Create administration user {username : admin , password : admin}
```

Comment:
This comment ({username : admin , password : admin}) because the password to hash form of expression, users noticed a username and password to log in to be.